src/circuit/gadgets.rs

```diff
@@ -213,17 +213,17 @@ mod tests {
     };
     use jf_utils::fr_to_fq;
 
+    type F = <Config as CapConfig>::ScalarField;
+    type JubjubParam = <Config as CapConfig>::JubjubParam;
```

Comment:
can we rename this to something else? This is BabyJubjub curve for our BN254 curve